.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+# use the shared Yast defaults
+inherit_from:
+  - /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+  - .rubocop_todo.yml
```

Comment:
I think it is better to merge stuff from todo here, so it is not spread over two files. At least this is what other modules do.

Reply:
The `.rubocop.yml` is your configuration, so what you want to change from the default configuration, and the `.rubocop_todo.yml` is what you want to enable, but didn't have enable yet. This todo file is used for avoiding having to solve many offenses when adding Rubocop to a big project. So this is perfectly fine.